README.md

```diff
@@ -198,3 +198,17 @@ If you find a security vulnerability or any security related issues,
 please DO NOT file a public issue, instead send your report privately to
 `security@coredns.io`. Security reports are greatly appreciated and we
 will publicly thank you for it.
+
+
+### Coordination of security releases with Kubernetes project
+
+As CoreDNS is now a feature part of Kubernetes, we are organizing our security release process 
```

Comment:
Which is nice and all, but does that more it more special than, say, Cloudflare which also use coredns in some project?

Reply:
As you propose below, let's make something more generic with "collaborating projects"